tests/test_chathistory_mongo.sh

```diff
@@ -0,0 +1,69 @@
+﻿#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+ip_address=$(hostname -I | awk '{print $1}')
+
+export MONGO_HOST=${ip_address}
+export MONGO_PORT=27017
+export DB_NAME=${DB_NAME:-"Conversations"}
+export COLLECTION_NAME=${COLLECTION_NAME:-"test"}
+
+function build_docker_images() {
+    cd $WORKPATH
+    echo $(pwd)
+    docker run -d -p 27017:27017 --name=mongo mongo:latest
+
+    docker build --no-cache -t opea/chathistory-mongo-server:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/chathistory/mongo/docker/Dockerfile .
+}
+
+function start_service() {
+
+    docker run -d --name="chathistory-mongo-server" -p 6013:6013 -p 6012:6012 -p 6014:6014 -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e no_proxy=$no_proxy -e MONGO_HOST=${MONGO_HOST} -e MONGO_PORT=${MONGO_PORT} -e DB_NAME=${DB_NAME} -e COLLECTION_NAME=${COLLECTION_NAME} opea/chathistory-mongo-server:latest
```

Comment:
--name=xxx -> --name=test-comps-xxx

Reply:
There are steps to clean up containers with "test-comps-" prefix in the CI workflow, so even the test failed, the CI can clean up the docker containers and release the port.

Reply:
Agreed, Done